app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,8 @@
     <string name="error_wallet_not_found_title">No wallet found</string>
     <string name="error_wallet_not_found_message">Try account or use an another phone number</string>
     <string name="error_too_often_otp_requests_message">You request SMS too often. Try again in %1d sec.</string>
+    <string name="error_shares_does_not_matches_title">Shares aren\'t\nmatch</string>
+    <string name="error_shares_does_not_matches_message">So sorry, baby\n\nIf you will write us, use error code #%1d</string>
```

Comment:
Really? Baby?? Are you sure it's correct?

Reply:
@1sakov do you see what i see?
![image](https://user-images.githubusercontent.com/99185280/192468008-8d63c7dd-48de-4adc-a06f-ff9fc7e1ab21.png)

Reply:
fixed with Nina